### PR TITLE
KevS-SK1267-Creation-How-to-create-subdomains-For-DomainDNS

### PR DESCRIPTION
**Pull request by Kazarus4**
FRFR done, need peer review and eventually a tech review.

### CODE REVIEW

**Review by BrenKt — APPROVED** [review]
Review done

File: pages/web_cloud/domains/domain_create_subdomains/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
+title: "Comment créer un sous-domaine ?"
+excerpt: "Découvrez la définition d'un sous-domaine et comment en créer chez OVHcloud"
+updated: 2023-11-09
```
Comment (Kazarus4) [inline pages/web_cloud/domains/domain_create_subdomains/guide.fr-fr.md]:
```suggestion
updated: 2023-11-24
```

File: pages/web_cloud/domains/domain_create_subdomains/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+---
+
+## Objectif <a name="goal"></a>
+
+Internet est composé de *serveurs* et d'*appareils* qui interagissent entre eux via un réseau global. Lorsque ces *serveurs* et ses *appareils* sont connectés au réseau Internet, une *adresse IP publique* (équivalent à une adresse postale) leur est attribuée. Cette *adresse IP* permet de joindre à distance un serveur ou un appareil, ainsi, un utilisateur est en mesure de consulter un site web en saisissant cette *adresse IP* grâce à son navigateur Internet installé sur son ordinateur.
+
+Les **noms de domaine** ont été instaurés pour faciliter l'accès à un site web pour les utilisateurs du réseau Internet. En effet, il est plus simple de retenir un nom composé d'une chaîne de caractères choisis (exemple: ovhcloud.com), plutôt qu'une suite de chiffres que compose une *adresse IP* (exemple: 54.39.46.56).
```
Comment (Kazarus4) [inline pages/web_cloud/domains/domain_create_subdomains/guide.fr-fr.md]:
```suggestion
Les **noms de domaine** ont été instaurés pour faciliter l'accès à un site web pour les utilisateurs du réseau Internet. En effet, il est plus aisé de retenir un nom composé d'une chaîne de caractères choisis (exemple: ovhcloud.com), plutôt qu'une suite de chiffres que compose une *adresse IP* (exemple: 54.39.46.56).
```

File: pages/web_cloud/domains/domain_create_subdomains/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+> La zone DNS active de votre nom de domaine n'est pas obligatoirement gérée auprès du même fournisseur que votre nom de domaine.
+>
+> 1 : Pour identifier l'endroit où se trouve la zone DNS active d'un nom de domaine enregistré chez OVHcloud, vous pouvez vous aider de notre guide « [Modifier les serveurs DNS d'un nom de domaine OVHcloud](/pages/web_cloud/domains/dns_server_general_information) ».
+>
+> 2 : Si votre nom de domaine n'est pas enregistré chez OVHcloud, rapprochez-vous du *bureau d'enregistrement* actuel de votre nom de domaine afin de connaître l'endroit où est hébergée sa zone DNS active.
+>
+
+Si les serveurs DNS déclarés pour votre nom de domaine ont l'une des deux formes suivantes :
+
+- *dnsXX.ovh.net* et *nsXX.ovh.net* (ou chacun des « X » représente un chiffre);
```
Comment (Kazarus4) [inline pages/web_cloud/domains/domain_create_subdomains/guide.fr-fr.md]:
```suggestion
- *dnsXX.ovh.net* et *nsXX.ovh.net* (où chacun des « X » représente un chiffre);
```

File: pages/web_cloud/domains/domain_create_subdomains/guide.fr-ca.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
+---
+title: "Comment créer un sous-domaine ?"
+excerpt: "Découvrez la définition d'un sous-domaine et comment en créer chez OVHcloud"
+updated: 2023-11-24
```
Comment (Kazarus4) [inline pages/web_cloud/domains/domain_create_subdomains/guide.fr-ca.md]:
```suggestion
updated: 2023-11-27
```

File: pages/web_cloud/domains/domain_create_subdomains/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
+---
+title: "Comment créer un sous-domaine ?"
+excerpt: "Découvrez la définition d'un sous-domaine et comment en créer chez OVHcloud"
+updated: 2023-11-24
```
Comment (Kazarus4) [inline pages/web_cloud/domains/domain_create_subdomains/guide.fr-fr.md]:
```suggestion
updated: 2023-11-27
```

**Pull request merged**